test/UndockedRegFreeWinRT/URFW_WinAppSDK/Test.testdef

```diff
@@ -0,0 +1,11 @@
+﻿{
+    "Tests": [
+        {
+            "Description": "UndockedRegFreeWinRT (URFW) tests (arm64 not currently enabled)",
```

Comment:
why not arm64?

Reply:
Ummm, copy pasta? But with mushrooms and basil! :P

```
DIR /S/B *.testdef
```

test\AppLifecycle\AppLifecycle.testdef
"Architectures": ["x64", "arm64"],
Why no x86?

test\DynamicDependency\Powershell\DynamicDependency_Powershell.testdef
"Architectures": ["x64", "arm64"],
Why no x86?

test\DynamicDependency\Test_Win32\DynamicDependency_API_Win32.testdef
"Architectures": ["x64", "arm64"],
Why no x86?

test\DynamicDependency\Test_WinRT\DynamicDependency_API_WinRT.testdef
"Architectures": ["x64", "arm64"],
Why no x86?

test\PackageManager\API\Test.testdef
"Architectures": ["x86", "x64"],
Why no arm64?

test\PushNotificationTests\PushNotificationTests.testdef
"Architectures": ["x64"],
"Status": "Disabled"
Why no x86 or arm64?
And why disabled?

I'll take care of DynamicDependency + PackageManager (and URFW). You got the rest?